# The control flag for whether rplidar driver should drop this frame of scan.
bool allow_publishing
